lib/listener-tracker.js

```diff
@@ -79,7 +89,16 @@ EventEmitterWrapper.prototype = {
             listener.apply(this, arguments);
         };
 
-        this._target.once(event, wrappedListener);
+        if (this._target.once) {
+          this._target.once(event, wrappedListener);
+        } else {
+          // need to save this so we can remove it below
+          var onceListener = function() {
```

Comment:
this was a little tricky since there is no `once` for window, document, etc. But we can kind of fake it, but to remove the listener after the first event, we need a reference to it.